app/views/questions/onboarding.html.erb

```diff
@@ -0,0 +1,30 @@
+<h1>test</h1>
+
+
+
+<div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
```

Comment:
It's likely going to be easier to use something that comes close already rather than reshaping a bootstrap carousel which would require quite a lot of adjustment.

Have a look at this one, for instance:
https://codepen.io/fcc-danielw/pen/RwGbvmO